wikipedia-sentences-0.8.2/app.py

```diff
@@ -0,0 +1,59 @@
+__version__ = '0.0.1'
+
+import os
+import sys
+
+from jina.flow import Flow
+
+num_docs = int(os.environ.get('MAX_DOCS', 500))
+
+def config():
+    parallel = 2 if sys.argv[1] == 'index' else 1
+    shards = 2
+
+    os.environ['JINA_PARALLEL'] = str(parallel)
```

Comment:
Rename these as `JINA_SHARDS_smth` (Where smth is the pod name) and use the `shards` prop. in the yml

Reply:
This is the deprecated version. I'll be removing it once we've confirmed the `0.9.21` version works.